FSharp.Core.Nuget/FSharp.Core.nuspec

```diff
@@ -114,5 +114,9 @@
     <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.optdata" target="lib\netstandard1.6\FSharp.Core.optdata" />
     <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.sigdata" target="lib\netstandard1.6\FSharp.Core.sigdata" />
     <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.xml" target="lib\netstandard1.6\FSharp.Core.xml" />
+    <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.optdata" target="runtimes\any\native\FSharp.Core.optdata" />
```

Comment:
Because nuget picks up only dlls from the lib folder, other stuff is explected to live in content/runtime folder.

Maybe @KevinRansom can give some more insight?